### ISSUE TITLE
Hi there, could you plz share the parameters from your experiment?

### ISSUE THREAD

**Issue by xxalisonn** [jzystc/MetaP#2]
Hi there, thanks for the code :) 
I failed to reproduced the similar result as ones in your paper with parameters below:
dataset: NELL-One
data_form: Pre-Train
few: 1
num_query: 3
metric: Hits@10
learning_rate: 0.001
step: train
mask_ratio: 0
fine_tune: True
rum: True
vbm: True
aggregator: max

The best results I can reproduced right now are(which were way lower than what you posted):
Hit@10 | Hit@5 | Hit@1 | MRR
0.308 | 0.237 | 0.118 | 0.181

So I wonder if there were something wrong with my parameter settings. Could you plz share the parameters you used for your best results?  It would help A LOT.
